src/ngraph/frontend/onnx_import/utils/broadcasting.cpp

```diff
@@ -66,10 +40,25 @@ static std::vector<ngraph::Shape> get_numpy_broadcast_shape(ngraph::Shape left_s
 static std::pair<ngraph::Shape, std::vector<ngraph::Shape>>
     get_numpy_broadcast_shapes(const ngraph::NodeVector& inputs)
 {
-    auto shape_left_fold = [](const ngraph::Shape& accumulator,
+    auto shape_left_fold = [](ngraph::Shape& accumulator,
```

Comment:
When operating on a non-const reference to the accumulator, you're left padding the internal object held as an accumulator by std::accumulate. I think it would be safer, especially without unit tests, to pass it to this labda by value.